### PR TITLE
Updated Documentation for pytest-splunk-addon v130 

### PR DESCRIPTION
**Pull request by zahrasidhpuri-crest**
As a part of this PR, the following has been updated:

- Added details about the **breaker** parameter in **Data Generator** Section.

- Updated **How To Use** section.

- Updated **Index Time Tests** Section.

### CODE REVIEW

File: docs/index_time_tests.rst [hunk 1]
```diff
@@ -122,16 +122,16 @@ Test Scenarios
...
+      `expected_event_count` which is calculated by pytest-splunk-addon from the `sample_count` 
+      parameter provided in the pytest-splunk-addon-data.conf.
 
     **Workflow:**
 
     * Generates an SPL query using sourcetype and host from the event. 
     * Execute the SPL query in a Splunk instance.
     * The number of results obtained from the search query is matched with the 
-      *expected_event_count* parameter. Both the values must be same.
+      *expected_event_count* value, which is calculated by the plugin. Both the values must be same.
```
Comment (bhargav-nariyani-crest) [inline docs/index_time_tests.rst:134]: Looks ambiguous.

File: docs/how_to_use.rst [hunk 2]
```diff
@@ -66,6 +66,8 @@ The tool assumes the Splunk Add-on is located in a folder "package" in the proje
 
 .. note::
    If live events are available in external Splunk instance or docker splunk, then SA-Eventgen is not required.
+   This is applicable only for Knowledge objects tests and CIM compatibility tests.
+   For the execution of Index Time tests, the data has to be ingested using pytest-splunk-addon.
```
Comment (bhargav-nariyani-crest) [inline docs/how_to_use.rst:70]: Need to update this version wise.

Comment (zahrasidhpuri-crest) [reply docs/how_to_use.rst]: Updated

File: docs/index_time_tests.rst (hunk 1 above)
Comment (zahrasidhpuri-crest) [reply docs/index_time_tests.rst]: Updated

**Pull request merged**